src/js/common/schemaform/FormApp.jsx

```diff
@@ -68,8 +71,7 @@ export default class FormApp extends React.Component {
           {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
           {content}
         </div>
-        <div className="usa-width-one-third medium-4 columns show-for-medium-up">
-        </div>
+        <div className="columns usa-width-one-third medium-4"><AskVAQuestions/></div>
```

Comment:
I'm with @jbalboni. We already have access to `formConfig`, so it'd be super easy to do something like:
```jsx
const contactInfo = typeof formConfig.contactInfo === 'function'
  ? formConfig.contactInfo
  : null;

...
  {formConfig.title && !isIntroductionPage && <FormTitle title={formConfig.title} subTitle={formConfig.subTitle}/>}
  {content}
</div>
{contactInfo}
<span className="js-test-location hidden" data-location={trimmedPathname} hidden></span>
...
```
But with hopefully a better name than `contactInfo` :man_shrugging: